client/blindai/client.py

```diff
@@ -300,3 +335,8 @@ def close_connection(self):
             self.channel = None
             self.stub = None
             self.policy = None
+
+    def export_proof(self):
+        """Dump the proof of execution to a json file"""
+        with open("execution_proof.json", "w") as proof_file:
```

Comment:
Maybe the docstring can be updated a bit here, explaining the kind of data that will be exported.